### ISSUE TITLE
jni多线程

### ISSUE THREAD

**Issue by akeopa** [VomPom/flutter_ffi_tutorial#1]
c++里面多线程调用有奔溃

**Comment by VomPom** [comment]
尊敬的先生/女士：
					
				
				
					
						
							   您好！您发送的邮件我已经收到，我回尽快回复您。   谢谢!

						
					
				
				
				
					
						
							

								
									--王鹏（落叶挽歌）

**Comment by VomPom** [comment]
再贴一下你的code：

    void f1(int n) {
    static JniEnv sync_env;
    ……
    }

    FFI_EXPORT int platform_sync(int value1, int value2) {
    thread t(f1, 8);
    return 1;
    }

thread t(f1, 8);
return 1;

这样子的操作是不被允许的（不仅仅是jni，在任何c++环境下执行都是），应该加入 t.join() 或者 t.detach()操作，直到线程的执行被完成再return
